tests.py

```diff
@@ -7,80 +7,86 @@
     text_box_search_folder, text_box_search, snippet_search_folder
 
 
-def positive_test_find_expressive_term():
+def positive_test_dictionary_search(): # pylint: disable=invalid-name
+    """Positive test for find_expressive_term"""
+    tree, _ = prepare_tree('database/Chopin.xml')
+    assert text_box_search(tree, "Expressive Terms", 'cresc.') == text_box_search(tree, "Expressive Terms", 'crescendo'),\
+        "text_box_search : crescendo and cresc. returning different results"
+
+
+def positive_test_find_expressive_term(): # pylint: disable=invalid-name
     """Positive test for find_expressive_term"""
     _, root = prepare_tree('database/Chopin.xml')
     element_et_list = find_expressive_term(root, 'legatissimo')
-    assert len(element_et_list), "find_expressive_term: legatissimo not found."
+    assert element_et_list, "find_expressive_term: legatissimo not found."
 
 
 def positive_test_find_artic():
     """Positive test for find_artic"""
     _, root = prepare_tree('database/Chopin.xml')
     element_artic_list = find_artic(root, 'stacc')
-    assert len(element_artic_list) != 0, "find_artic: stacc not found"
+    assert element_artic_list, "find_artic: stacc not found"
 
 
 def positive_test_search():
     """Positive test for search"""
     tree, _ = prepare_tree('database/Chopin.xml')
-    input_xml = etree.parse('testinput.xml')
-    input_root = input_xml.getroot()
-    measure_match_list = search(input_root, tree)
-    assert len(measure_match_list) != 0, "search: unsuccessful"
+    input_tree, _ = prepare_tree('testinput.xml') # pylint: disable=c-extension-no-member
+    measure_match_list = search(input_tree, tree)
+    assert measure_match_list, "search: unsuccessful"
 
 
-def positive_test_get_attrib_from_element():
+def positive_test_get_attrib_from_element(): # pylint: disable=invalid-name
```

Comment:
Since testing functions tend to have long names, it's better to disable the pylint `invalid-name` test on the entire file. Put it as a comment at the top, ie.

```python
 """tests.py contains a test for every back-end algorithm"""

# pylint: disable=invalid-name

...
```